src/edu/washington/escience/myria/api/encoding/DatasetEncoding.java

```diff
@@ -23,4 +25,5 @@
   public Boolean importFromDatabase;
   public List<List<IndexRef>> indexes;
   public Boolean overwrite;
+  public PartitionFunction howPartitioned = new RoundRobinPartitionFunction(null);
```

Comment:
So maybe just call it partitionFunction? I was trying to use the same name for every relevant field.